ModelicaTest/Blocks.mo

```diff
@@ -1310,32 +1362,40 @@ if homotopy is active, the solution accepted by the assert statement (x = 100) i
-        points={{-59,110},{-50,110},{-50,-40},{-42,-40}}, color={0,0,127}));
-    connect(cosine.y, to_day.u) annotation (Line(
-        points={{-59,110},{-50,110},{-50,-70},{-42,-70}}, color={0,0,127}));
-    connect(cosine.y, to_hour.u) annotation (Line(
-        points={{-59,110},{-50,110},{-50,-100},{-42,-100}}, color={0,0,127}));
-    connect(cosine.y, to_minute.u) annotation (Line(
-        points={{-59,110},{-50,110},{-50,-130},{-42,-130}}, color={0,0,127}));
-    connect(cosine.y, to_litre.u) annotation (Line(
-        points={{-59,110},{-50,110},{-50,128},{28,128},{28,110},{38,110}}, color={0,0,127}));
-    connect(cosine.y, to_kWh.u) annotation (Line(
-        points={{-59,110},{-50,110},{-50,128},{28,128},{28,80},{38,80}}, color={0,0,127}));
     connect(to_litre.y, from_litre.u) annotation (Line(
-        points={{61,110},{68,110}}, color={0,0,127}));
+        points={{71,110},{78,110}}, color={0,0,127}));
     connect(to_kWh.y, from_kWh.u) annotation (Line(
-        points={{61,80},{70,80}}, color={0,0,127}));
-    connect(cosine.y, to_bar.u) annotation (Line(
-        points={{-59,110},{-50,110},{-50,128},{28,128},{28,50},{38,50}}, color={0,0,127}));
-    connect(cosine.y, to_gps.u) annotation (Line(
-        points={{-59,110},{-50,110},{-50,128},{28,128},{28,20},{38,20}}, color={0,0,127}));
+        points={{71,70},{78,70}}, color={0,0,127}));
     connect(to_bar.y, from_bar.u) annotation (Line(
-        points={{61,50},{70,50}}, color={0,0,127}));
+        points={{71,30},{78,30}}, color={0,0,127}));
     connect(to_gps.y, from_gps.u) annotation (Line(
-        points={{61,20},{70,20}}, color={0,0,127}));
+        points={{71,-10},{78,-10}}, color={0,0,127}));
+    connect(cosine1.y, to_degF.u)
+      annotation (Line(points={{-59,80},{-42,80}}, color={0,0,127}));
+    connect(cosine2.y, to_degRk.u)
+      annotation (Line(points={{-79,50},{-42,50}}, color={0,0,127}));
+    connect(cosine3.y, to_deg.u)
+      annotation (Line(points={{-59,20},{-42,20}}, color={0,0,127}));
+    connect(cosine4.y, to_rpm.u)
+      annotation (Line(points={{-79,-10},{-42,-10}}, color={0,0,127}));
+    connect(cosine5.y, to_kmh.u)
+      annotation (Line(points={{-59,-40},{-42,-40}}, color={0,0,127}));
+    connect(cosine6.y, to_day.u)
+      annotation (Line(points={{-79,-70},{-42,-70}}, color={0,0,127}));
+    connect(cosine7.y, to_hour.u)
+      annotation (Line(points={{-59,-100},{-42,-100}}, color={0,0,127}));
+    connect(cosine8.y, to_minute.u)
+      annotation (Line(points={{-79,-130},{-42,-130}}, color={0,0,127}));
+    connect(to_litre.u, cosine9.y)
+      annotation (Line(points={{48,110},{41,110}}, color={0,0,127}));
+    connect(cosine12.y, to_gps.u)
+      annotation (Line(points={{41,-10},{48,-10}}, color={0,0,127}));
+    connect(cosine11.y, to_bar.u)
+      annotation (Line(points={{41,30},{48,30}}, color={0,0,127}));
+    connect(cosine10.y, to_kWh.u)
+      annotation (Line(points={{41,70},{48,70}}, color={0,0,127}));
```

Comment:
Minor comment: Can I please ask for sorting by cosine instance name and having the cosine connector as first argument of the connect statement?